static/home/js/signprot-multitabtable2.js

```diff
@@ -0,0 +1,1029 @@
+/*global yadcf*/
+/*eslint complexity: ["error", 8]*/
+/*eslint wrap-iife: ["error", "outside"]*/
+/*eslint quotes: ["error", "double", { "avoidEscape": true }]*/
+let oTable1 = [];
+let oTable2 = [];
+
+
+var tableToExcel = (function () {
```

Comment:
Note to all devs: this tableToExcel code is now reproduced more than 10 times. Time to make this a function generic

Reply:
Where would these generic functions go?
The reason why I copied it was that if I imported existing JS's, there would be non-generic function clashing with what I have. Say, if I imported it from browser_functions.js some of the selection functions there would clash with "mine", such as, select_all.